jscomp/test/as_inline_record_test.js

```diff
@@ -6,10 +6,18 @@ function getName(t) {
   return t.renamed;
 }
 
+function getName$p(t) {
+  return t.name;
```

Comment:
you can see the error here, should be:
```suggestion
  return t.renamed;
```